docs/sections/use-case-requirements.html

```diff
@@ -10,30 +10,230 @@ <h3>Use Cases and Requirements</h3>
+
+<p>
+These endpoints do not require the application to be authenticated:
+</p>
+
+<pre class="example">
+https://platform.example/organization/123/presentations/available
+https://platform.example/organization/123/presentations/submissions
+</pre>
+
+</section>
+
+<section>
+  <h5>assertionMethod</h5>
+<p>
+The set of supported <code>assertionMethod</code> DID URLs for the organization.
+</p>
+
+<p>
+See <a href="https://www.w3.org/TR/did-core/#assertion">assertionMethod</a>.
+</p>
+</section>
+
+<section>
+  <h5>authentication</h5>
+<p>
+The set of supported <code>authentication</code> DID URLs for the organization.
+</p>
+
+<p>
+See <a href="https://www.w3.org/TR/did-core/#authentication">authentication</a>.
+</p>
+</section>
+
+ <p>
+  The <code>didDocument</code> MAY contain a <code>verificationMethod</code> section,
+  which MAY be used to support <code>did:web</code> based verification relationships.
+ </p>
+<p>
+  The <code>didDocument</code> MUST NOT contain <code>verificationMethods</code> where the controller is different from the DID Subject.
+</p>
+
+<p>
+  All <code>Ed25519VerificationKey2018</code> types support <code>Ed25519Siganture2018</code>.
+</p>
+
+<p>
+  All <code>JsonWebKey2020</code> types support <code>JsonWebSignature2020</code> AND <code>VC-JWT</code>.
+</p>
+
+<p>
+  In this example, the organization suports authentication and credential issuance with the same two keys, identified via the DID URLs in the relationships:
+</p>
+
+<pre class="example">
+  {
+    "@context": [
+      "https://www.w3.org/ns/did/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "did:web:platform.example:organization:123",
+    "alsoKnownAs": [
+      "https://platform.example/organization/123",
+      "did:key:z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V",
+      "did:key:z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi"
+    ],
+    "assertionMethod": [
+      "did:key:z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V#z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V",
+      "did:key:z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi#z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi"
+    ],
+     "authentication": [
+      "did:key:z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V#z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V",
+      "did:key:z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi#z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi"
+    ],
+    "service": [
+      {
+        "id": "did:web:platform.example:organization:123#traceability-api",
+        "type": "TraceabilityAPI",
+        "serviceEndpoint": "https://platform.example/organization/123"
+      }
+    ]
+  }
+</pre>
+
+<p class="advisement">
+Implementations are encouraged to require authentication for all endpoints.
+</p>
+
+<p class="advisement">
+Implementations are encouraged to `allow-list` specific DIDs per `CLIENT_ID`.
+</p>
+
```

Comment:
This advisement should be near the discussion of `CLIENT_ID` (see relocation above), not as far removed as possible...
```suggestion
<p class="advisement">
Implementations are encouraged to `allow-list` specific DIDs per `CLIENT_ID`.
</p>

```

Reply:
I can't accept this change here now, because I accepted your other suggestion, and there is no longer anything to change. resolving.